syntaxes/Dafny.tmLanguage

```diff
@@ -54,7 +54,7 @@
 		</dict>
 		<dict>
 			<key>begin</key>
-			<string>(\w+)\s*\(</string>
+			<string>(\w+)\s*(&lt;[^&gt;]+&gt;)?\s*\(</string>
```

Comment:
Can you do this for signatures only?  I don't think it should happen in method bodies.

Reply:
Because in a method body it doesn't work consistently, e.g. `Foo<Bar<Baz>>(xyz)`

Reply:
I fixed it for depths until 3.